openlibrary/plugins/openlibrary/js/subjects.js

```diff
@@ -39,6 +41,28 @@ function urlencode(query) {
     return parts.join("&");
 }
 
+/* Should really use createElement() */
+function renderTag(tag, keyvals, child) {
```

Comment:
This should be done with createElement but I would rather not invest time beyond this minimal implementation as we our composition is moving to react @bfalling